Objects/longobject.c

```diff
@@ -3611,24 +3611,26 @@ long_richcompare(PyObject *self, PyObject *other, int op)
     Py_RETURN_RICHCOMPARE(result, 0, op);
 }
 
-static void
```

Comment:
You need to wrap this with a `Py_DEBUG` so that it doesn't warn as unused on release builds.